Demo/src/main/java/com/braintreepayments/demo/MainFragment.kt

```diff
@@ -0,0 +1,192 @@
+package com.braintreepayments.demo
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Card
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.navigation.fragment.findNavController
+import com.braintreepayments.api.core.PaymentMethodNonce
+
+@Suppress("TooManyFunctions")
+class MainFragment : BaseFragment() {
+
+    private var nonce: PaymentMethodNonce? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return ComposeView(requireContext()).apply {
+            setContent {
+                Column(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                ) {
+
+                    Card(
+                        modifier = Modifier.padding(12.dp),
+                    ) {
+                        Text(
+                            modifier = Modifier
+                                .align(Alignment.CenterHorizontally)
+                                .padding(top = 8.dp),
+                            text = "PayPal",
+                            style = MaterialTheme.typography.headlineSmall
+                        )
+
+                        PaymentModuleButton(R.string.paypal_button) { launchPayPal() }
+                        PaymentModuleButton(R.string.shopper_insights_button) { launchShopperInsights() }
+                        PaymentModuleButton(R.string.paypal_messaging_button) { launchPayPalMessaging() }
+                        Spacer(modifier = Modifier.height(4.dp))
+                    }
+
+                    Card(
+                        modifier = Modifier.padding(12.dp),
+                    ) {
+                        Text(
+                            modifier = Modifier
+                                .align(Alignment.CenterHorizontally)
+                                .padding(top = 8.dp),
+                            text = "Other",
```

Comment:
I tried to group the PayPal modules since "Shopper Insights" is clearly defined as a PayPal module.

I put everything else in the same "Other" category. Is there a better way to group the remaining modules?

Reply:
I think either makes sense. Or if we don't find the groupings helpful we can remove then and just list PayPal first? I don't feel strongly either way.